eng/testing/tests.targets

```diff
@@ -72,6 +74,19 @@
                        OutputPath="$(RunScriptOutputPath)" />
 
     <Exec Condition="'$(TargetOS)' != 'Windows_NT'" Command="chmod +x $(RunScriptOutputPath)" />
+
+    <!-- Generate Mono .rsp file -->
+    <ItemGroup>
+      <MonoRspFiles Include="$(MonoProjectRoot)netcore\CoreFX.issues.rsp" />
+      <MonoRspFiles Include="$(MonoProjectRoot)netcore\CoreFX.issues_mac.rsp" Condition="'$(TargetOS)' == 'OSX'" />
+      <MonoRspFiles Include="$(MonoProjectRoot)netcore\CoreFX.issues_linux.rsp" Condition="'$(TargetOS)' == 'Linux'" />
+      <MonoRspFiles Include="$(MonoProjectRoot)netcore\CoreFX.issues_windows.rsp" Condition="'$(TargetOS)' == 'Windows_NT'" />
+    </ItemGroup>
+    <ItemGroup>
+      <MonoRspFileContent Include="$([System.IO.File]::ReadAllText(%(MonoRspFiles.Identity)))" />
```

Comment:
Should this be conditioned to only copy the rsp file when targeting Mono?

Reply:
We can do that, sure.

Reply:
I think it'd be better if the files were located under src/libraries

Reply:
We agreed on using Attributes instead. Is it feasible to remove these rsp files all together now?

Reply:
Not now, we still have to annotate many tests (100's+) with different attributes